src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -215,46 +137,27 @@
                   </v-card>
                 </div>
                 <div v-else-if="isProprietor">
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation as the Proprietor, enter the name
-                    or incorporation number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form. All other types of business cannot
-                    be the Proprietor.
+                  <p class="mb-0">
+                    Enter an existing B.C. business as the proprietor.
                   </p>
                 </div>
                 <div v-else-if="isPartner">
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation as a partner, enter the name or
-                    incorporation number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form. All other types of business cannot
-                    be a partner.
-                  </p>
-                  <HelpContactUs class="mt-6" />
-                </div>
-                <div v-else>
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation, enter the name or incorporation
-                    number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form.
+                  <p class="mb-0">
+                    Enter an existing B.C. business as a partner.
                   </p>
+
                 </div>
 
                 <BusinessLookup
                   :showErrors="enableRules"
                   :businessLookup="inProgressBusinessLookup"
                   :BusinessLookupServices="BusinessLookupServices"
+                  :label="businessLookupLabel"
```

Comment:
Why not hard-code this instead of using a variable/property?

Reply:
Because we are calling `BusinessLookup` from `'@bcrs-shared-components/business-lookup'`, and we set the previous label as a prop cause we do not want to effect other UIs.

Reply:
Are you saying this instance of BusinessLookup (in this Vue file) is used by other filings or other UIs?

Reply:
I think it refers to what you said here? If I don't misunderstand
![image](https://github.com/bcgov/business-create-ui/assets/144159934/e029a6b8-ab7b-4e21-8ee5-102bdc06423c)

Reply:
The BusinessLookup component is used in multiple places, but this specific instance of it (in RegAddEditOrgPerson.vue) is used only by firm registrations, so if you don't need different labels here then you can hard-code it.

(One exception to hard-coding a static string is if it's too long to put in the template; then we sometimes declare a variable for it in the script block.)

Reply:
Got it!! Thanks Severin, sorry for misunderstanding "hard-code" at here😥

Reply:
LGTM